frontend/asset_mapper.rst

```diff
@@ -233,8 +233,11 @@ If you want to download the package locally, use the ``--download`` option:
     $ php bin/console importmap:require bootstrap --download
 
 This will download the package into an ``assets/vendor/`` directory and update
-the ``importmap.php`` file to point to it. You *should* commit this file to
-your repository.
+the ``importmap.php`` file to point to it. You can either commit this directory
+to your repository, or ignore the directory in your ``.gitignore`` config file.
+If the directory is ignored, you can run the
```

Comment:
Is it « can » or « must » ?

Reply:
I think in 6.4, we should change the workflow from "must commit" to "should ignore". Let's rework the language for that - it'll be just like `composer install`: it's a "must" (assuming you are downloading assets locally).

In fact, would you mind opening a new recipe PR for AssetMapper 6.4 that includes `assets/vendor/` in `.gitignore`?

Reply:
While merging I did some tweaks according to your comments. See https://github.com/symfony/symfony-docs/commit/5909e6b5e6bcd5d4d61c4aa69d18139688ee00ee